install/kubernetes/helm/istio/values.yaml

```diff
@@ -40,6 +40,21 @@ global:
     # If set, newly injected sidecars will have core dumps enabled.
     enableCoreDump: false
 
+    # Default port for Pilot agent health checks. A value of 0 will disable health checking.
+    statusPort: 15003
+
+    # Default HTTP path for readiness probes on the statusPort.
+    readinessPath: /healthz/ready
+
+    # The initial delay for readiness probes in seconds.
+    readinessInitialDelaySeconds: 2
```

Comment:
@costinm @rshriram @mandarjog what do we think of these default values?

Reply:
still too high.. think of knative.. 2s is a long time for functions. Ours should be less than 500ms

Reply:
AFAICT [this is an integer is k8s](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#probe-v1-core), so the lowest we can go is 1. Is that reasonable?